app/controllers/admin/orders_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
       @orders = User.find_by(email: params[:email]).orders.not_InProgress
       flash[:notice] = 'No orders found' if @orders.empty?
     else
-      @orders = Order.not_InProgress
+      @orders = Order.includes(:address).not_InProgress
```

Comment:
in ruby, method names should not have capital letters

Reply:
fixed